src/main/resources/openapi/sqs/vca-api-sqs-messaging-erop.yaml

```diff
@@ -1,9 +1,9 @@
 openapi: 3.0.0
 info:
-  title: Voter Card Application API SQS Message Types for EROP Clients
```

Comment:
I couldn't see any obvious reason why this file needed to be VAC specific- is that true? (as I look at this PR I see some more VAC/VCA references which I'll remove once confirmed this is fine..!)

Reply:
This file is a copy of one that lives in the VCA repository - that repo is the "owner" of this file and it defines all of the queues that are "owned" by the VCA API. So the VAC/VCA references are expected :) In particular, the field being called `voterCardApplicationId` is what vca-api expects on the message listener.

Now that postal should be supported (and soon, proxy and OE), the new queue should be added to the SQS YAML file in the postal repo and this repo will need a copy of it... One of the many "conveniences" that comes from this microservice architecture!